internal/formats/spdx22json/decoder_test.go

```diff
@@ -61,6 +63,12 @@ func TestSPDXJSONDecoder(t *testing.T) {
 			f, err := os.Open("test-fixtures/spdx/" + test.path)
 			assert.NoError(t, err)
 
+			if !test.fail {
+				err = Format().Validate(f)
+				require.NoError(t, err)
+				_, err = f.Seek(0, io.SeekStart)
```

Comment:
Do we need to reset the offset here because Validate manipulates the file?

Reply:
it was necessary because it reads the file, to Dan's point I noticed this test doesn't help the encoder, so I removed it.